src/components/common/CopyButton/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { faCopy } from '@fortawesome/free-regular-svg-icons'
+import { faCheck } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+import { DISPLAY_TEXT_COPIED_CHECK } from 'apps/explorer/const'
+
+const Icon = styled(FontAwesomeIcon)<{ copied: boolean }>`
+  color: ${({ theme, copied }): string => (copied ? theme.green1 : theme.icon)};
```

Comment:
@biocom u had some concerns with the theming. Is this subject settled?

Reply:
Not a concern, but would say that we'd likely benefit from having a single `green` color variable throughout. I know that right now we still support other UI's/codebase, but just stating as a goal.

Reply:
ok, although in this case I'm referring to some concerns about the theming system and the template literals. I mean, do we have consensus in how we style our theme?

Reply:
To be honest I haven't dug to deep yet into how @W3stside setup theming. AFAIK we were using this method above Leandro implemented. But leaving this to David to correct where I'm off.

Reply:
okok, if u have no issues, then the matter is settled. 2 weeks ago, u expressed some concerns about the theme, so I was referring to that.

Reply:
My concern there was specifically how we were writing the media queries :)

Reply:
ahh, right! only for media queries. OK. Are u still concern about that? should be settle it?

Reply:
Yes, we should. Forgot to mention in our last sync. I will write on our channel.